lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -115,7 +115,7 @@ def submit(actions)
         elsif RETRYABLE_CODES.include?(status)
           @logger.info "retrying failed action with response code: #{status} (#{error})"
           actions_to_retry << action
-        else
+        elsif not skip_errors.include?(error["type"])
```

Comment:
it feels weird to mix "error" and the log level is 'warn'. Maybe we can rename this from 'error' to 'failure' ?